ci_framework/roles/rhol_crc/tasks/main.yml

```diff
@@ -43,47 +45,61 @@
     - not cifmw_rhol_crc_force_cleanup|bool
     - not cifmw_rhol_crc_reuse | bool
 
-- name: Manage RHOL/CRC if not existing
-  when: (cifmw_rhol_crc_force_cleanup|bool) or not crc_present|bool
+- name: Clean/deploy CRC
   block:
-    - name: Check RHOL/CRC binary exists
-      ansible.builtin.stat:
-        path: "{{ cifmw_rhol_crc_binary }}"
-      register: cifmw_rhol_crc_binary_stat
+  - name: Retrieve RHOL/CRC if not existing
```

Comment:
missing indentation I'd say. Must be reflected to the whole block content, so that we get:
```YAML
- name: foo
  block:
    - name: bar
      block:
        - name: baz
```